### ISSUE TITLE
The screen freezes

### ISSUE THREAD

**Issue by MarkusCicero** [bvschaik/julius#641]
Hi! I noticed such a problem when playing Julius. When a lot of sounds are being played, the screen starts to freeze a bit.
This is especially noticeable during attacks: when an enemy unit appears on the map, an alarm sound is played. And if you use a cheat code to increase the number of attackers, then more signaling sounds will be played. At this point, the screen freezes especially hard.
Could it be because of the compatibility of my audio card?
The parameters of my computer are as follows:
Windows 7 64-bit, AMD FX 6300 six-core 3.50 GHz processor, 4 Gb RAM, NVIDIA GeForce GT 730 video card, C-Media CMI8738 PCI Audio card

**Comment by bvschaik** [comment]
That's weird, never heard that before. It sounds like the CPU is too busy at that point.
Could you try turning off the sounds from the Options -> Sound options menu in the game and see if the screen still freezes? If the sounds are off, the game doesn't try to play them, so we can either confirm or exclude the sound card as the cause.

**Comment by MarkusCicero** [comment]
Turned off all sounds through the game menu. I created a lot of attackers with a cheat code and the game really worked fine. But as soon as I turned on the sounds, the screen began to freeze again. Turned off all sounds - and everything is fine again.
The model of my audio card is very old, and the drivers for it are from 2011. It looks like Julius uses more modern libraries. Because there were no such problems in the original game.
